cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -580,6 +580,25 @@ function LogViewerController($scope, $window, LogViewerStore, myLogsApi, LOGVIEW
     checkForScrollbar();
   };
 
+  vm.LogLevelsFallThrough = function(levelObj, eventType) {
```

Comment:
this function does not need to be attached to `vm`. This can be a private method.

also nit: lowercase `log`